src/ops/euclid.rs

```diff
@@ -174,6 +182,14 @@ pub trait CheckedEuclid: Euclid {
     /// Finds the euclid remainder of dividing two numbers, checking for underflow, overflow and
     /// division by zero. If any of that happens, `None` is returned.
     fn checked_rem_euclid(&self, v: &Self) -> Option<Self>;
+
+    /// Returns both the checked euclidian division quotien and remainer
+    ///
+    /// By default, it internaly calls both `Euclid::div_euclid` and `Euclid::rem_euclid`,
+    /// but it can be overwritten in order to implement some optimization.
```

Comment:
```suggestion
    /// By default, it internally calls both `Euclid::checked_div_euclid` and `Euclid::checked_rem_euclid`,
    /// but it can be overridden in order to implement some optimization.
```

Reply:
done